MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,20 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onLoadError: Observer<String?>?
```

Comment:
`onLoadError` could be named `onErrorStateChange` following the `onLoadingStateChange` property. The current name might be misleading since the `Observer` is notified not only when a `LoadError` occurs, but also to show or dismiss any type of error message.
```suggestion
	var onErrorStateChange: Observer<String?>?
```